flink-table-store-core/src/test/java/org/apache/flink/table/store/file/operation/TestCommitThread.java

```diff
@@ -173,7 +173,11 @@ public void run() {
             } catch (Exception e) {
                 throw new RuntimeException(e);
             }
-            writer.close();
+            try {
+                writer.close();
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
```

Comment:
Remove `try {.writer.sync() }` above this code fragment because it is included in `writer.close()`.